greenmail-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -14,15 +14,15 @@
     </context-param>
     <context-param>
         <param-name>greenmail.smtp.port</param-name>
-        <param-value>10025</param-value>
+        <param-value>1025</param-value>
     </context-param>
     <context-param>
-        <param-name>greenmail.pop3.host</param-name>
+        <param-name>greenmail.imap.host</param-name>
```

Comment:
with this one, I don't know how this servlet will behave if you wanted to use context.xml - does it check for empty value and goes on trying other options?